ares/urls.py

```diff
@@ -3,8 +3,10 @@
 """
 from django.conf.urls import url, include
 from django.contrib import admin
+from django.conf import settings
+from django.conf.urls.static import static
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url('^accounts/', include('accounts.urls')),
-]
+] + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
you're right. it's ok for now. I'll merge this now but I'm also opening a security issue about this that must be resolved before the first deployment.

and I like the nginx way. but It doesn't really differentiate.